src/app/views/login/LoginForm.jsx

```diff
@@ -31,7 +31,7 @@ export default class LoginForm extends Component {
         return (
             <div className="grid grid--justify-center">
                 <div className="grid__col-3">
-                    <h1 className="text-center">Login</h1>
+                    <h1 className="line-height--46">Log in to your Florence account</h1>
```

Comment:
Surely you want to make this change more globally? The page title should be consistent across all pages.

Reply:
Great line-height info. I have made this relative to the font size now and across all H1s I have checked numerous screens that use H1s in Florence and they all look fine.